### PR TITLE
Adding role property for aria:ErrorList

### PR DESCRIPTION
**Pull request by divdavem**

### CODE REVIEW

File: src/aria/widgets/CfgBeans.js [hunk 1]
```diff
@@ -1675,8 +1675,12 @@ module.exports = Aria.beanDefinitions({
                 },
                 "ariaLive" : {
                     $type : "json:Boolean",
-                    $description : "Specifies whether role=alert and aria-live=assertive are included (true) or omitted (false) on the container element (defaults to true, only used if waiAria is true)",
+                    $description : "Specifies whether the aria-live=assertive property is included (true) or omitted (false) on the container element (defaults to true, only used if waiAria is true)",
                     $default : true
+                },
+                "role": {
+                    $type : "json:String",
+                    $description : "Specifies the role to use on the container of the ErrorList, if any. It defaults to 'status' if ariaLive is true. It is only used if waiAria is true."
```
Comment (fbasso) [inline src/aria/widgets/CfgBeans.js:1683]: Maybe you should specify what the defaut if ariaLive is false (no role).

**Comment by fbasso** [comment]
Thanks for your PR @divdavem !

I just added a small comment regarding the doc, otherwise, it's fine for me !

File: src/aria/widgets/CfgBeans.js (hunk 1 above)
Comment (divdavem) [reply src/aria/widgets/CfgBeans.js:1683]: @fbasso Thank you for your comment. Yes, indeed, I have to mention the behaviour when `ariaLive` is false.

**Comment by divdavem** [comment]
@fbasso Thank you for your review. As discussed, I have changed the wording of the documentation to better explain.

**Comment by divdavem** [comment]
@fbasso Thank you!

**Pull request merged**